tools/vsimporter/src/vswriter/VSTemplateProject.cpp

```diff
@@ -157,9 +159,45 @@ static bool isAppxManifestFileName(const String& fileName) {
     return 0 == fileName.compare(fileName.length() - extension.length(), String::npos, extension);
 }
 
-static void insertUrlSchemes(const String& file, const StringSet& schemes) {
-    //
-    // Inject registered URL schemes into the AppX manifest file, in this format:
+static std::vector<String> getCacheFolders(const String& entitlementsPath) {
+    Plist::dictionary_type* pDict = nullptr;
+    boost::any pAny;
+    try {
+        Plist::readPlist(entitlementsPath.c_str(), pAny);
```

Comment:
>Plist::readPlist(entitlementsPath.c_str(), pAny); [](start = 8, length = 49)

Needs to be relative to xcodeproj or something, not relative to the current dir.